examples/rtl8720dn/wioetrminal.go

```diff
@@ -0,0 +1,79 @@
+//go:build wioterminal
```

Comment:
Check spelling of the name of this file, please.

Reply:
I made a mistake.
Fixed and force-pushed.